### PR TITLE
Display the results of the last build for each test

### PR DESCRIPTION
**Pull request by dandenisa**

### CODE REVIEW

File: controller/static/app/projects/edit.controller.js [hunk 1]
```diff
@@ -941,5 +959,55 @@
...
+            var buildIdsForATest = [];
+            for (var test in vm.res.testResults) {
+                if (vm.res.testResults[test].testId === testid) {
+                    buildIdsForATest.push(vm.res.testResults[test].buildId);
+                }
+            }
+            vm.latestBuildId = buildIdsForATest[buildIdsForATest.length - 1];
+        }
+
+        function getTestBuildResults(projectId, testId, buildId) {
```
Comment (AndreeaDora) [inline controller/static/app/projects/edit.controller.js]: @dandenisa Please remove the method if we are not using it

File: controller/static/app/projects/edit.html [hunk 2]
```diff
@@ -1008,6 +1011,72 @@ <h4 class="ui header">Tests Details &nbsp &nbsp &nbsp
     </div>
 </div>
 
+<div id="view-test-results-modal" class="ui fullscreen modal transition view results">
+
+
```
Comment (AndreeaDora) [inline controller/static/app/projects/edit.html]: @dandenisa Please add the close icon and fix the functionality so the modal window can be closed when we press Esc

File: controller/static/app/projects/edit.controller.js [hunk 3]
```diff
@@ -128,6 +135,16 @@
             return text;
         }
 
+        $(".ui.fullscreen.modal.transition.view.results").on('keyup', function (evt) {
+            if (evt.keyCode == 27) {
+                $(".ui.fullscreen.modal.transition.view.results").modal('hide');
+            }
+        });
+
+        $("#inspect-view-test-results").on('click', function () {
```
Comment (AndreeaDora) [inline controller/static/app/projects/edit.controller.js]: @dandenisa The modal window is not being closed when we press the magnifying glass to see the image vulnerabilities

**Comment by dandenisa** [comment]
another pull request with these changes has been opened

**Pull request closed without merging**